src/components/listing/VideoCard.jsx

```diff
@@ -36,9 +51,26 @@ const VideoCard = ({ video }) => {
         />
         {showOptions && (
           <ul className={`${styles.options} br-s`}>
-            <li className="px-xs my-xs pointer">
-              <small> Watch Later</small>
-            </li>
+            {!watchlater.find((item) => item._id === video._id) ? (
```

Comment:
You can create a utility function to check the condition, as you will be using the same for other features also.

Reply:
Good suggestion. Will do this while finishing the project up.